extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java

```diff
@@ -45,4 +53,46 @@ public void testJsonResourceNotFound() {
                 .contentType(ContentType.JSON);
     }
 
+    @Test
+    public void shouldDisplayNewAddedFileIn404ErrorPage() {
+        String CONTENT = "html content";
+        test.addResourceFile(META_INF_RESOURCES + "index2.html", CONTENT);
+
+        RestAssured.get("/index2.html")
```

Comment:
Is any sort of wait needed here between adding the resource file and invoking the `GET /index2.html` to ensure that dev mode has processed the new file?

Reply:
Hi @aguibert. No, it is not needed. Changes will be picked detected and picked automatically as indicated in the line 63 and 64.